parlai/crowdsourcing/tasks/model_chat/task_config/image_model_opts.yaml

```diff
@@ -0,0 +1,3 @@
+transresnet_multimodal: >
```

Comment:
what is the `>` for? and is there a reason this isn't setup the normal yaml way? like `model_file: <mf>`

Reply:
Ah, the `>` allows you to specify a block of text. The reason I have the params set up like this is so I can parse the args using ParlaiParser, which I generally try to use as much as possible to avoid edge cases from differences in how params are parsed on the command line vs. when passing in an `Opt`. (It's related to the point you make above about some params having a `dist` specified, i.e. there are subtle differences in the flag depending on how it's used.) I also think it's useful to keep the normal command-line syntax here so that users can just specify commands exactly how they'd do it on the command line, to minimize confusion

Reply:
yeah that's fine with me